packages/kit-bg/src/services/ServicePrime/ServicePrime.tsx

```
@@ -0,0 +1,558 @@
+  async promptPrimeLoginEmailDialog() {
+    // eslint-disable-next-line no-async-promise-executor
+    const email = await new Promise<string>(async (resolve, reject) => {
+      const promiseId = this.backgroundApi.servicePromise.createCallback({
+        resolve,
+        reject,
+      });
+      await primeLoginDialogAtom.set((v) => ({
+        ...v,
+        promptPrimeLoginEmailDialog: promiseId,
+      }));
+    });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)